### DISCUSSION TITLE
[Q/A] What power mode is used with SX1262, LDO or DC-DC ?

### DISCUSSION THREAD

**Discussion by joyel24** [meshtastic/firmware#5234] [Q&A] [answered]
I'm asking this especially in receive mode as it can eventually increase the battery life significantly in relay or router mode.

![IMG_20241103_050315.jpg](https://github.com/user-attachments/assets/1b38cb5a-f16f-4948-8a88-78f248267fe3)

**Comment by caveman99** [comment, marked as answer]
DC-DC
